### ISSUE TITLE
Metadata ETS:  Using gmx schemas

### ISSUE THREAD

**Issue by MarieLambois** [inspire-eu-validation/ets-repository#28]
It seems that the validator refuses metadata using gmx schemas. 
 
First issue any type inheriting CharacterString is refused : 
The dataset has 1 file(s) with errors for this assertion.
XML document 'metadonnee_csw.xml': The file has 4 schema validation error(s).
XML document 'metadonnee_csw.xml': 51:129: cvc-complex-type.2.4.a: Invalid content was found starting with element '{"http://www.isotc211.org/2005/gmx":FileName}'. One of '{"http://www.isotc211.org/2005/gco":CharacterString}' is expected.
XML document 'metadonnee_csw.xml': 132:133: cvc-complex-type.2.4.a: Invalid content was found starting with element '{"http://www.isotc211.org/2005/gmx":FileName}'. One of '{"http://www.isotc211.org/2005/gco":CharacterString}' is expected.
XML document 'metadonnee_csw.xml': 167:141: cvc-complex-type.2.4.a: Invalid content was found starting with element '{"http://www.isotc211.org/2005/gmx":Anchor}'. One of '{"http://www.isotc211.org/2005/gco":CharacterString}' is expected.
XML document 'metadonnee_csw.xml': 201:157: cvc-complex-type.2.4.a: Invalid content was found starting with element '{"http://www.isotc211.org/2005/gmx":Anchor}'. One of '{"http://www.isotc211.org/2005/gco":CharacterString}' is expected.

Second issue the XML validator is not able to find the metadata root if stated as gmx:MD_Metadata 
cvc-elt.1.a: Cannot find the declaration of element 'gmd:MD_Metadata'.

[XMLissue.zip](https://github.com/inspire-eu-validation/ets-repository/files/760814/XMLissue.zip)

**Comment by cportele** [comment]
Yes, currently the Abstract Test Suite is quite strict and basically prohibits the use of subtypes. In particular, 
* all XPath expressions are "relative to gmd:MD_Metadata"
* validation is against gmd only, the gmx namespace is not supported (http://inspire.ec.europa.eu/id/ats/metadata/1.3/xml-encoding/schema-validation, based on section 2.1.2 in the Technical Guidance)

I agree that at least subtypes defined in the 19139 schemas including gmx should be allowed, but this should be confirmed by the MIG subgroup responsible for the ATSs.

cc: @michellutz

**Comment by michellutz** [comment]
Sorry for getting back to this only now. I think the issues raise the larger question on how we deal with issues coming from feedback on the validator/ETS, which are not ETS-implementation related, but related to issues in the ATS or even TGs. I think there could be a role for a revived MIWP-5 sub-group that would collect and discuss such issues and would elaborate change requests to ATS/TGs to be reviewed and endorsed by the MIG (similar to the way the MIWP-14 sub-group works with change requests coming from the Thematic Clusters). We will discuss this at the upcoming MIG-T face-to-face meeting.

**Comment by michellutz** [comment]
I will also provide my own view on the issues - but after I'm back from my holidays (13 March).
